Clava-JS/src-api/clava/memoi/MemoiUtils.ts

```diff
@@ -0,0 +1,149 @@
+/**
+ * The C data types the memoization instrumentation library can handle.
+ */
+export enum MemoiDataType {
+  INT = "INT",
+  DOUBLE = "DOUBLE",
+  FLOAT = "FLOAT",
+}
+
+export default class MemoiUtils {
```

Comment:
Can set mathFunctionSigs be derived from MathHInfo.ts?

Reply:
Yes and no. The information could perhaps be retrieved using that API but this is also a whitelist.